src/app/components/game/game.component.ts

```diff
@@ -68,6 +68,14 @@ export class GameComponent implements OnInit, OnChanges {
         color: string;
         gA: number;
     }[];
+    private arrows: {
```

Comment:
awesome object, this is great, and looks great in the UI - could we refactor this name to be something like `drawnArrows` though, I think it would be more clear / specific